cli/tests/cli-reference@.md.snap

```diff
@@ -2227,6 +2227,8 @@ Show commit description and changes in a revision
 
 Cryptographically sign a revision
 
+This command will fail, if no signing backend is configured (see https://jj-vcs.github.io/jj/latest/config/#commit-signing).
```

Comment:
(Optional) I think we currently tend to format links like:

```
This command requires configuring a [commit signing] backend.

[commit signing]: https://jj-vcs.github.io/jj/latest/config/#commit-signing
```

You can of course adjust the wording further.

Reply:
Thanks! I wasn't aware of that convention and only sampled one or two commands for their help text, which did not have links in them.